src/main/java/in/twizmwaz/cardinal/module/modules/score/ScoreModule.java

```diff
@@ -71,11 +71,21 @@ public void unload() {
     }
 
     public int getScore() {
```

Comment:
Well, i just let it like that, because it's only really used in the scoreboard, or chat, where it always displays as an int, also, i don't think all arround the code we should use "setScore(getScore() + 1)", and call the score event, it's way better to "addScore(something)", and if it's different than the old socre, it calls the socre event